extensions/amp-story-captions/0.1/amp-story-captions-presets.css

```diff
@@ -0,0 +1,48 @@
+@import '../../amp-story/1.0/amp-story-shadow-reset.css';
+
+/* default */
+
+.amp-story-captions-default .amp-story-captions-cue-wrapper {
+  font-size: calc(2.5 * var(--story-page-vh, 8px)) !important;
+  font-family: 'Roboto', sans-serif !important;
```

Comment:
Will this use Helvetica on iOS?

Reply:
[Yes](https://fontsarena.com/blog/operating-systems-default-sans-serif-fonts/)

<img width="400" alt="Screen Shot 2022-04-18 at 11 42 39 AM" src="https://user-images.githubusercontent.com/3860311/163841643-aa7c0197-bdea-4f7d-ba2a-fe895e0ef0d0.png">